# Datasets

flu-crew datasets used in manuscripts, tutorials, and other work.

## h1-1C-riskAssessment

Phylogenetic analysis files for:
Le Sage, V., Souza, C.K., Rockey, N.C., Shepard, M., Zanella, G.C., Arruda, B., Wang, S., Drapeau, E., Doyle, J.D., Xu, ., Barbeau, D.J., Paulson, J.C., McElroy, A.K., Hensley, S.E., Anderson, T.K., Baker, A.L., and Lakdawala, S.S. Eurasian 1C swine influenza A virus exhibits high pandemic risk traits.


## h3-2019-spillover

Phylogenetic analysis files for:
Powell, J.D., Thomas, M.N., Anderson, T.K., Zeller, M.A., Gauger, P.C., and Vincent Baker, A.L. A 2018-2019 human seasonal H3N2 influenza A virus spillover into swine with demonstrated virus transmission in pigs.

We gratefully acknowledge all data contributors, i.e., the Authors and their Originating laboratories responsible for obtaining the specimens, and their Submitting laboratories for generating the genetic sequence and metadata and sharing via the GISAID Initiative, on which components of this research is based. The GISAID acknowledgment table is provided in this repo.


## h5n1-hpai

Variant call output for:
Arruda, B., Vincent Baker, A.L., Buckley, A., Torchetti, M., Hines Bergeson, N., Killian, M.L., Lantz, K., Anderson, T.K. Avian and mammalian isolates of highly pathogenic avian influenza belonging to the goose/Guangdong 2.3.4.4b hemagglutinin phylogenetic clade infect swine and mammalian isolates transmit to limited contacts.

Raw FASTQ files for samples are available at FigShare - for immediate access please email tavis.anderson@usda.gov:
URL pending.

The consensus genomes for the four strains are provided at NCBI Genbank or are hosted at GISAID. We gratefully acknowledge all data contributors, i.e., the Authors and their Originating laboratories responsible for obtaining the specimens, and their Submitting laboratories for generating the genetic sequence and metadata and sharing via the GISAID Initiative, on which components of this research is based. The GISAID acknowledgment table is provided in this repo.

A/red fox/Michigan/22-018712-001-original/2022  = EPI_ISL_15078253 <br>
A/raccoon/Washington/22-018406-002-original/2022  = EPI_ISL_15078252 <br>
A/turkey/Minnesota/22-010654-002-original/2022 = EPI_ISL_16555202 <br>
A/bald eagle/Florida/W22-134-CL/2022 = OP470733-OP470740 <br>
A/bald eagle/Florida/W22-134-OP/2022 = OP377436-OP377443 <br>

## h1n2-pandemic_risk

Tree files and data for:

Le Sage, V., Rockey, N.C.,  McCarthy, K.R., French, A.J., Shephard, M.J.,Jones, J.E., Walter, S.G., Doyle, J.D., Xu, L., Barbeau, D.J., Frizzell, S.A., Myerburg, M.M., McElroy, A.K., Anderson, T.K., Vincent Baker, A.L., Lakdawala, S.S. Potential pandemic risk of circulating swine H1N2 influenza viruses.

## h1-swinetoferret

Tree files and data for:

Kimble, J.B.; Souza, C.K.; Anderson, T.K.; Arendsee, Z.W.; Hufnagel, D.E.; Young, K.M.; Lewis, N.S.; Davis, C.T.; Thor, S.; Vincent Baker, A.L. Interspecies Transmission from Pigs to Ferrets of Antigenically Distinct Swine H1 Influenza A Viruses with Reduced Reactivity to Candidate Vaccine Virus Antisera as Measures of Relative Zoonotic Risk. [Viruses 2022, 14(11), 2398. DOI: 10.3390/v14112398](https://pubmed.ncbi.nlm.nih.gov/36366493/).

## h3-np-reassortment

Figures and data for: 

Thomas, M.N.; Zanella, G.C.; Cowan, B.; Caceres C.J.; Rajao, D.S.; Perez, D.R.; Gauger, P.C.; Baker, A.L.; Anderson, T.K. Nucleoprotein reassortment enhanced transmissability of H3 1990.4.a clade influenza A virus in swine. (manuscript in preparation)

## h3-swinetoferret

Tree files and data for:

Souza, C.K., Kimble, J.B., Anderson, T.K., Arendsee, Z.W., Hufnagel, D.E., Young, K.M., Gauger, P.C., Lewis, N.S., Davis, C.T., Thor, S., Vincent Baker, A.L. Transmission from swine-to-ferret of antigenically drifted contemporary swine H3N2 influenza A virus as indications of zoonotic risk to humans. [Viruses 2023, 15(2), 331. DOI: 10.3390/v15020331](https://pubmed.ncbi.nlm.nih.gov/36851547/).
